src/lib/model/DFSPEnvConfigModel.js

```diff
@@ -26,25 +26,27 @@ class DFSPEnvConfigModel {
      * Gets DFSPs details from MCM Server
      *
      * @param opts {object}
-     * @param opts.envId {string}
      * @param opts.dfspId {string}
      */
     async getDFSPList(opts) {
-        const url = `/environments/${opts.envId}/dfsps`;
+        const url = `/dfsps`;
         return this._mcmServerRequest.get(url);
     }
 
+    async findStatus() {
+        return this._hubRequest.get(`/dfsps/${this._dfspId}/status`);
+    }
```

Comment:
Is this used anywhere? Was it intentionally in this PR?

Reply:
Yes, it's used in `mojaloop-payment-manager-management-api`